Codeinwp/templates-patterns-collection
# Fix/bug 81

### Summary
Re-map categories if placeholders are present.
References: Codeinwp/templates-patterns-collection#81

### Test instructions
Use the Gist here: https://gist.github.com/preda-bogdan/f4d1b4e876ecc9424e98b837dc8519b5

1. Install Neve on a new instance
2. Download the gist from above as .zip and install and activate as a plugin
3. Use the plugin build from this branch to test

**NOTE:** The plugin from GIST should already be active since we save the sites list inside a 12h transient.

The changes from from https://github.com/Codeinwp/demo-data-exporter/pull/78 are already present on the staging instance: https://demosites.io/


When importing the Digital Store template it should map the categories correctly.

It should replace all the block categories placeholders with the correct imported categories.
The format used is `replace_cat_%d`.


### Testing Instance example:
I went ahead and did all the steps above here:

```
 Admin area URL: https://test-staging-imp.tastewp.com/wp-admin 
 Username: bogdanpreda 
 Password: HCSC-l4aZqc 
```

You can use it as an example but please test on you own instances.
